sormas-api/src/test/java/de/symeda/sormas/api/utils/SizeGenerator.java

```diff
@@ -0,0 +1,105 @@
+ */
+
+package de.symeda.sormas.api.utils;
+
+import static de.symeda.sormas.api.utils.FieldConstraints.CHARACTER_LIMIT_SMALL;
+
+import java.lang.annotation.Annotation;
+import java.util.List;
+
+import javax.validation.constraints.Size;
+
+import org.junit.Assert;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Utf8;
+
+import de.symeda.sormas.api.EntityDto;
+import de.symeda.sormas.api.caze.CaseDataDto;
+import de.symeda.sormas.api.clinicalcourse.ClinicalVisitDto;
+import de.symeda.sormas.api.contact.ContactDto;
+import de.symeda.sormas.api.event.EventDto;
+import de.symeda.sormas.api.event.EventParticipantDto;
+import de.symeda.sormas.api.immunization.ImmunizationDto;
+import de.symeda.sormas.api.person.PersonDto;
+import de.symeda.sormas.api.report.AggregateReportDto;
+import de.symeda.sormas.api.report.WeeklyReportDto;
+import de.symeda.sormas.api.sample.AdditionalTestDto;
+import de.symeda.sormas.api.sample.PathogenTestDto;
+import de.symeda.sormas.api.sample.SampleDto;
+import de.symeda.sormas.api.task.TaskDto;
+import de.symeda.sormas.api.therapy.PrescriptionDto;
+import de.symeda.sormas.api.therapy.TreatmentDto;
+import de.symeda.sormas.api.visit.VisitDto;
+import uk.co.jemos.podam.api.ObjectStrategy;
+import uk.co.jemos.podam.api.PodamFactory;
+import uk.co.jemos.podam.api.PodamFactoryImpl;
+import uk.co.jemos.podam.api.PodamUtils;
+import uk.co.jemos.podam.api.RandomDataProviderStrategyImpl;
+
+public class SizeGenerator {
+
+	private final static Class<? extends EntityDto>[] dtoClasses = new Class[] {
+		CaseDataDto.class,
+		ImmunizationDto.class,
+		ContactDto.class,
+		PersonDto.class,
+		EventDto.class,
+		EventParticipantDto.class,
+		SampleDto.class,
+		PathogenTestDto.class,
+		AdditionalTestDto.class,
+		TaskDto.class,
+		VisitDto.class,
+		WeeklyReportDto.class,
+		AggregateReportDto.class,
+		PrescriptionDto.class,
+		TreatmentDto.class,
+		ClinicalVisitDto.class
+	};
+
+	public static void main(String[] args) throws Exception {
+		final PodamFactory factory = new PodamFactoryImpl();
+
+		RandomDataProviderStrategyImpl strategy = (RandomDataProviderStrategyImpl) factory.getStrategy();
+		strategy.addOrReplaceAttributeStrategy(Size.class, new ObjectStrategy(){
+			@Override
+			public Object getValue(Class<?> attrType, List<Annotation> attrAnnotations) {
+				long length = CHARACTER_LIMIT_SMALL;
+
+				StringBuilder sb = new StringBuilder();
+				while (sb.length() < length) {
+					sb.append(PodamUtils.getNiceCharacter());
+				}
+				return sb.toString();
+			}
+		});
+		factory.setStrategy(strategy);
+
+		for (Class<? extends EntityDto> dtoClass : dtoClasses) {
+			getSizeOfFilledDto(factory, dtoClass);
+		}
+	}
+
+	private static void getSizeOfFilledDto(PodamFactory factory, Class<? extends EntityDto> dtoClass) throws JsonProcessingException {
+		final EntityDto dto = factory.manufacturePojo(dtoClass);
```

Comment:
Do you take compression factor into account? If not, that needs to be added because it reduces the size of the DTOs when they are transfered to the mobile app.